src/main/kotlin/com/chattriggers/ctjs/api/message/TextComponent.kt

```diff
@@ -252,6 +262,26 @@ class TextComponent : Text {
         Message(this).actionBar()
     }
 
+    private fun parseItemContent(obj: Any?): HoverEvent.ItemStackContent? {
+        return when (obj) {
+            is ItemStack -> obj
+            is Item -> obj.toMC()
+            is String -> ItemType(obj).asItem().toMC()
+            is HoverEvent.ItemStackContent -> return obj
+            else -> null
```

Comment:
Let's throw an error in the else case instead of quietly doing nothing, same for parseEntityContent